ProcessMessage/handler.ts

```diff
@@ -422,6 +432,55 @@ export const getProcessMessageHandler = ({
                 rejection_reason: RejectionReasonEnum.USER_NOT_FOUND,
                 status: RejectedMessageStatusValueEnum.REJECTED
               }),
+              TE.chain(() =>
+                pipe(
+                  // setting TTL to 3 years for message-status entries
+                  lMessageStatusModel.updateTTLForAllVersions(
+                    [newMessageWithoutContent.id],
+                    TTL_FOR_USER_NOT_FOUND
+                  ),
+                  TE.mapLeft((error: CosmosErrors) => {
+                    telemetryClient.trackEvent({
+                      name: "api.messages.create.failstatusttlset",
```

Comment:
A little more readable

```suggestion
                      name: "api.messages.create.fail-status-ttl-set",
```